tests/integration/cli_resource_test.go

```diff
@@ -0,0 +1,158 @@
+//go:build integration
+
+package integration
+
+import (
+	"crypto/ed25519"
+
+	"github.com/cheqd/cheqd-node/tests/integration/cli"
+	"github.com/cheqd/cheqd-node/tests/integration/network"
+	"github.com/cheqd/cheqd-node/tests/integration/testdata"
+	cli_types "github.com/cheqd/cheqd-node/x/cheqd/client/cli"
+	"github.com/cheqd/cheqd-node/x/cheqd/types"
+	"github.com/google/uuid"
+	"github.com/multiformats/go-multibase"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("cheqd cli", func() {
+	It("can create diddoc, create resource, query it, query all resource versions of the same resource name, query resource collection", func() {
+		// Create a new DID Doc
+		collectionId := uuid.NewString()
+		did := "did:cheqd:" + network.DID_NAMESPACE + ":" + collectionId
+		keyId := did + "#key1"
+
+		pubKey, privKey, err := ed25519.GenerateKey(nil)
+		Expect(err).To(BeNil())
+
+		pubKeyMultibase58, err := multibase.Encode(multibase.Base58BTC, pubKey)
+		Expect(err).To(BeNil())
+
+		payload := types.MsgCreateDidPayload{
+			Id: did,
+			VerificationMethod: []*types.VerificationMethod{
+				{
+					Id:                 keyId,
+					Type:               "Ed25519VerificationKey2020",
+					Controller:         did,
+					PublicKeyMultibase: string(pubKeyMultibase58),
+				},
+			},
+			Authentication: []string{keyId},
+		}
+
+		signInputs := []cli_types.SignInput{
+			{
+				VerificationMethodId: keyId,
+				PrivKey:              privKey,
+			},
+		}
```

Comment:
Same as above. Would a table be a less verbose way of describing this?

Reply:
Same as above, since container nodes change declaration scope.